apps/content/migrator/migrations/6/index.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose')
+const { appConfiguration } = require('../../../config')
+const TENANT = process.env.TENANT || '0'
+
+const Configuration = mongoose.model('Configuration')
+
+/**
+ * update site configuration
+ */
+
+/**
+ * check if app-configuration doesn't have themeStylesUrl propery
+ */
+async function check () {
```

Comment:
"check" is too general IMO why not call the function hasMissingThemeConfig ? this way you can just return `!Configuration.countDocuments..` and the function won't need the async declaration cause we'll won't. use await, in future if you'll add more checks add a validateConfigs() function that will call hasMissingThemeConfig() and other checks but meanwhile you aint gonna need it :P

Reply:
If I understand your question correctly, you can't do that because check is a mandatory function in the migration process (you run check, if needed you run migrate and then verify it's results using the verify function). Also, since it's the only variable that changed in that version, you won't need to add any other checks here (maybe in the next migration when other variables will change).

Reply:
As Dror said, it's part of the migration process.
This draft PR aims to change part of the collections data structure, so it needs to provide a migration version with 3 methods:
check- to check if the current DB is correct or not.
migrate - the actual script to migrate the DB into the new structure.
verify - return true if the migration succeeded (the the DB again).

Reply:
Thanks I wasn't familiar with that process \m/